pages/examples/continuous-deployment/helm-basic-with-inline-values.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Deploy basic helm chart with inline values
+description: 'Learn how to deploy a Grafana instance using Helm chart with inline configuration values, including ingress setup and TLS configuration'
+---
+
+## Overview
+
+In this guide, you'll learn how to deploy Grafana (a popular monitoring tool) on your Kubernetes cluster using Plural.
+We'll show you how to use Helm charts - think of them as pre-packaged applications - with inline configuration values,
+which means we'll define all our settings directly in the deployment file. This approach makes it easy to manage and
+update your applications.
+
+## Prerequisites
```

Comment:
this technically needs an ingress controller/cert-manager + externaldns to make it accessible externally too

Reply:
(in this case it's going to the mgmt cluster so that's already there but might be worth calling out and mentioning they're installed by default with self-hosted management clusters)

Reply:
good point, I will add that

Reply:
Added. Can you recheck?